extras/asr-services/providers/vibevoice/transcriber.py

```
@@ -0,0 +1,393 @@
+        # DEBUG: Log actual output format for troubleshooting
+        logger.info(f"Raw output preview (first 500 chars): {raw_output[:500]}")
+        logger.info(f"Raw output preview (last 500 chars): {raw_output[-500:]}")
+
+        # Extract JSON array from assistant response
+        # Strategy: Find the outermost [ ] that contains valid JSON
+        # Look for array starting with [{ which indicates segment objects
+        json_match = re.search(r'\[\s*\{.*\}\s*\]', raw_output, re.DOTALL)
+
+        if not json_match:
+            logger.warning("Could not find JSON array in output, returning raw text only")
+            logger.warning(f"Output does not match pattern [{{...}}], checking for other formats...")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)